helpers/split-image

```diff
@@ -182,8 +182,8 @@ split_image() {
   disk2_img_asc=$(eib_outfile disk2.img.asc)
 
   local disk1_img_csum disk2_img_csum
-  disk1_img_asc=$(eib_outfile disk1.img.sha256)
```

Comment:
This is Right and Proper, though split image support is going away https://phabricator.endlessm.com/T30922 so it's a little moot.